package/cluster/network/definition.yaml

```diff
@@ -21,6 +21,40 @@ spec:
                 id:
                   type: string
                   description: ID of this Network that other objects will use to refer to it.
+                parameters:
+                  type: object
+                  properties:
+                    region:
+                      type: string
+                      default: us-west-2
+                      description: AWS region
```

Comment:
Same as above. Please mention the cloud provider documentation URL in the description to make it clear where the region list is coming from.